src/components/Changelog/IncidentChangelogItems.tsx

```diff
@@ -0,0 +1,31 @@
+import { IncidentHistory } from "../../api/services/IncidentsHistory";
+import { relativeDateTime } from "../../utils/date";
+import { Avatar } from "../Avatar";
+import IncidentHistoryItemTypeContent from "./IncidentHistoryItemTypeContent";
+
+type IncidentChangelogItemProps = {
+  history: IncidentHistory;
+};
+
+export default function IncidentChangelogItem({
+  history
+}: IncidentChangelogItemProps) {
+  return (
+    <li className="mb-8 ml-6" key={history.id}>
```

Comment:
```suggestion
    <li className="mb-1 ml-1" key={history.id}>
```

Reply:
The suggested margin is too small, but I have adjusted it.